src/renderer/helpers/api/local.js

```diff
@@ -223,15 +223,19 @@ export async function getLocalVideoInfo(id) {
     return info
   }
 
-  if (hasTrailer) {
-    /** @type {import('youtubei.js').YTNodes.PlayerLegacyDesktopYpcTrailer} */
+  if (hasTrailer && info.playability_status.status !== 'OK') {
+    /** @type {import('youtubei.js').YTNodes.PlayerLegacyDesktopYpcTrailer | import('youtubei.js').YTNodes.YpcTrailer} */
     const trailerScreen = info.playability_status.error_screen
 
     const trailerInfo = new Mixins.MediaInfo([{ data: trailerScreen.trailer.player_response }])
```

Comment:
Could you please change this to a call to `info.getTrailerInfo()` as that should work for both trailer node types with your YouTube.js changes.